MetersPanel.c

```
@@ -138,6 +139,7 @@ static HandlerResult MetersPanel_eventHandler(Panel* super, int ch) {
       case '-':
          Vector_moveUp(this->meters, selected);
          Panel_moveSelectedUp(super);
```

Cppcheck (2.20.0)

[style] The function 'isNonnegative' is never used.

(unusedFunction)